src/main/scala/com/atomist/tree/marshal/LinkedJsonTreeDeserializer.scala

```diff
@@ -103,6 +102,21 @@ object LinkedJsonTreeDeserializer extends LazyLogging {
     }
 }
 
+class EmptyLinkableContainerTreeNode() extends ContainerTreeNode {
+
+  override def nodeName: String = "empty"
+
+  override def value: String = ""
```

Comment:
Null is un-Scala. However, where we interface to JavaScript, it's a different story.

So in this case I would return null and document in Scala that the behavior is not what might be expected and maybe add a non-`@Export` `valueAsOption` or the like.

Empty string is not an out of band value.